docs/Streams.md

```diff
@@ -79,6 +79,14 @@ The application can optimize its processing for that case but should be ready to
 
 When the buffer count is 0, it signifies the reception of a QUIC frame with empty data, which also indicates the end of stream data.
 
+## Summary - Common handling of receive data events
+
+If the application...
```

Comment:
There should be one more case mentioned: returning SUCCESS but reducing TotalBufferLength to indicate no more data can be processed right now, which pauses further receives.